src/wallet/test/rpc_wallet_tests.cpp

```diff
@@ -156,7 +156,7 @@ BOOST_AUTO_TEST_CASE(rpc_wallet)
     BOOST_CHECK_THROW(CallRPC("getbalance " + keyIO.EncodeDestination(demoAddress)), runtime_error);
 
     /*********************************
-     * 			listunspent
+     *              listunspent
```

Comment:
These tab/space changes aren't necessary; either revert them or make all of these comments consistent.

Reply:
Ok, I made them all consistent. That seemed better than trying to revert the right part of the right commit. Plus they were inconsistent before so it fixes that, and I would just run into the same problem next time I try to edit this file anyway.